lib/internal/fs/rimraf.js

```diff
@@ -22,27 +22,23 @@ const {
 const { join } = require('path');
 const { setTimeout } = require('timers');
 const notEmptyErrorCodes = new Set(['ENOTEMPTY', 'EEXIST', 'EPERM']);
+const retryErrorCodes = new Set(['EBUSY', 'EMFILE', 'ENOTEMPTY', 'EPERM']);
```

Comment:
```suggestion
const retryErrorCodes = new Set(['EBUSY', 'EMFILE', 'ENFILE', 'ENOTEMPTY', 'EPERM']);
```

à la https://github.com/nodejs/node/pull/30499?

Reply:
Done!